frontend/src/utility/Utils.ts

```diff
@@ -27,3 +27,7 @@ export function getRelativeTimeSince(time: number): string {
     }
```

Comment:
Can you update this to catch the 1 hours ago typo?

Reply:
Ah good catch